### ISSUE TITLE
Restarting VSCode using WSL  cannot connect to Github Copilot

### ISSUE THREAD

**Issue by areyeslo** [microsoft/vscode-copilot-release#3400]
Type: <b>Bug</b>

On windows 11 Enterprise version 23H2 Copilot stop working using WSL 2 with distro Ubuntu after restarting VSCode:

wsl distribution list:
PS C:\Users\ArturoRL> wsl --list
Windows Subsystem for Linux Distributions:
Ubuntu (Default)
docker-desktop
docker-desktop-data

Unregister Ubuntu:
PS C:\Users\ArturoRL> wsl --unregister Ubuntu
Unregistering.
The operation completed successfully.
PS C:\Users\ArturoRL>

Install WSL:
PS C:\Users\ArturoRL> wsl --install
Ubuntu is already installed.
Launching Ubuntu...
Installing, this may take a few minutes...
Please create a default UNIX user account. The username does not need to match your Windows username.
For more information visit: https://aka.ms/wslusers
Enter new UNIX username: areyeslo
New password:
Retype new password:
passwd: password updated successfully
Installation successful!
To run a command as administrator (user "root"), use "sudo <command>".
See "man sudo_root" for details.

Welcome to Ubuntu 24.04.1 LTS (GNU/Linux 5.15.167.4-microsoft-standard-WSL2 x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/pro

 System information as of Thu Dec 19 07:38:53 MST 2024

  System load:  0.25                Processes:             67
  Usage of /:   0.1% of 1006.85GB   Users logged in:       0
  Memory usage: 17%                 IPv4 address for eth0: 172.19.49.97
  Swap usage:   0%

This message is shown once a day. To disable it please create the
/home/areyeslo/.hushlogin file.

Set default version for the distributions:
PS C:\Users\ArturoRL> wsl --set-default-version 2
For information on key differences with WSL 2 please visit https://aka.ms/wsl2
The operation completed successfully.

List of WSL distributions:
PS C:\Users\ArturoRL> wsl --list
Windows Subsystem for Linux Distributions:
docker-desktop (Default)
docker-desktop-data
Ubuntu

PS C:\Users\ArturoRL> wsl --set-default Ubuntu
The operation completed successfully.

areyeslo@QLK333KY23473FB:~$ pwd
/home/areyeslo
areyeslo@QLK333KY23473FB:~$ mkdir Github
areyeslo@QLK333KY23473FB:~$ cd Github
areyeslo@QLK333KY23473FB:~/Github$ git clone https://github.com/bcgov/lcfs.git
Cloning into 'lcfs'...
remote: Enumerating objects: 34273, done.
remote: Counting objects: 100% (236/236), done.
remote: Compressing objects: 100% (114/114), done.
remote: Total 34273 (delta 194), reused 123 (delta 122), pack-reused 34037 (from 2)
Receiving objects: 100% (34273/34273), 27.28 MiB | 25.49 MiB/s, done.
Resolving deltas: 100% (25730/25730), done.

The VSCode is showing Welcome to Copilot and button Use Copilot.

The VSCode version is the following:
Version: 1.96.1 (user setup)
Commit: 42b266171e51a016313f47d0c48aca9295b9cbb2
Date: 2024-12-17T17:50:05.206Z
Electron: 32.2.6
ElectronBuildId: 10629634
Chromium: 128.0.6613.186
Node.js: 20.18.1
V8: 12.8.374.38-electron.0
OS: Windows_NT x64 10.0.22631

Then, go to Extensions and install Github Copilot - Install in WSL:Ubuntu

Installation
Identifier github.copilot
Version 1.253.0
Last Updated 2024-12-19, 07:49:42
Size 50.9 MB
Marketplace Published 2021-06-29, 08:26:17
Last Released 2024-12-18, 18:40:53

I get the usual Ask Copilot chat and it works.

After installation, the window showing showing Welcome to Copilot and button Use Copilot changed to Ask Copilot window.
In addition, under accounts I get another account areyeslo (Github Copilot Chat) in addition to the existing account areyeslo (Github).

Generate Debug configuration with Github Copilot
## Node setup

- Number of root certificates: 147
- Operating system: Linux
- Operating system version: 5.15.167.4-microsoft-standard-WSL2
- Operating system architecture: x64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

## Network Configuration

- Proxy host: n/a
- Proxy port: n/a
- Kerberos SPN: n/a
- Reject unauthorized: disabled
- Fetcher: HelixFetcher

## Reachability

- github.com: HTTP 200
- api.github.com: HTTP 200
- proxy.individual.githubcopilot.com: HTTP 200
- api.individual.githubcopilot.com: HTTP 200
- telemetry.individual.githubcopilot.com: HTTP 400

## VS Code Configuration

- HTTP proxy: 
- HTTP proxy authentication: n/a
- Proxy Strict SSL: true
- Extension HTTP proxy support: override

## Extensions

- Is `win-ca` installed?: false
- Is `mac-ca` installed?: false

## Authentication

- GitHub username: areyeslo

Github Copilot Chat Diagnostics:
## GitHub Copilot Chat

- Extension Version: 0.23.2 (prod)
- VS Code: vscode/1.96.1
- OS: Linux
- Remote Name: wsl

## Network

User Settings:
```json
  "github.copilot.advanced.debug.useElectronFetcher": true,
  "github.copilot.advanced.debug.useNodeFetcher": false,
  "github.copilot.advanced.debug.useNodeFetchFetcher": true
```

Connecting to https://api.github.com:
- DNS ipv4 Lookup: 140.82.114.6 (24 ms)
- DNS ipv6 Lookup: Error (30 ms): getaddrinfo ENOTFOUND api.github.com
- Proxy URL: None (0 ms)
- Electron fetch: Unavailable
- Node.js https: HTTP 200 (313 ms)
- Node.js fetch (configured): HTTP 200 (334 ms)
- Helix fetch: HTTP 200 (392 ms)

Connecting to https://api.individual.githubcopilot.com/_ping:
- DNS ipv4 Lookup: 140.82.114.21 (14 ms)
- DNS ipv6 Lookup: Error (17 ms): getaddrinfo ENOTFOUND api.individual.githubcopilot.com
- Proxy URL: None (0 ms)
- Electron fetch: Unavailable
- Node.js https: HTTP 200 (283 ms)
- Node.js fetch (configured): HTTP 200 (307 ms)
- Helix fetch: HTTP 200 (306 ms)

## Documentation

In corporate networks: [Troubleshooting firewall settings for GitHub Copilot](https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-firewall-settings-for-github-copilot).

areyeslo@QLK333KY23473FB:~/Github$ curl -v https://api.github.com
* Host api.github.com:443 was resolved.
* IPv6: (none)
* IPv4: 140.82.113.5
*   Trying 140.82.113.5:443...
* Connected to api.github.com (140.82.113.5) port 443
* ALPN: curl offers h2,http/1.1
* TLSv1.3 (OUT), TLS handshake, Client hello (1):
*  CAfile: /etc/ssl/certs/ca-certificates.crt
*  CApath: /etc/ssl/certs
* TLSv1.3 (IN), TLS handshake, Server hello (2):
* TLSv1.3 (IN), TLS handshake, Encrypted Extensions (8):
* TLSv1.3 (IN), TLS handshake, Certificate (11):
* TLSv1.3 (IN), TLS handshake, CERT verify (15):
* TLSv1.3 (IN), TLS handshake, Finished (20):
* TLSv1.3 (OUT), TLS change cipher, Change cipher spec (1):
* TLSv1.3 (OUT), TLS handshake, Finished (20):
* SSL connection using TLSv1.3 / TLS_AES_128_GCM_SHA256 / X25519 / id-ecPublicKey
* ALPN: server accepted h2
* Server certificate:
*  subject: CN=*.github.com
*  start date: Mar  7 00:00:00 2024 GMT
*  expire date: Mar  7 23:59:59 2025 GMT
*  subjectAltName: host "api.github.com" matched cert's "*.github.com"
*  issuer: C=GB; ST=Greater Manchester; L=Salford; O=Sectigo Limited; CN=Sectigo ECC Domain Validation Secure Server CA
*  SSL certificate verify ok.
*   Certificate level 0: Public key type EC/prime256v1 (256/128 Bits/secBits), signed using ecdsa-with-SHA256
*   Certificate level 1: Public key type EC/prime256v1 (256/128 Bits/secBits), signed using ecdsa-with-SHA384
*   Certificate level 2: Public key type EC/secp384r1 (384/192 Bits/secBits), signed using ecdsa-with-SHA384
* using HTTP/2
* [HTTP/2] [1] OPENED stream for https://api.github.com/
* [HTTP/2] [1] [:method: GET]
* [HTTP/2] [1] [:scheme: https]
* [HTTP/2] [1] [:authority: api.github.com]
* [HTTP/2] [1] [:path: /]
* [HTTP/2] [1] [user-agent: curl/8.5.0]
* [HTTP/2] [1] [accept: */*]
> GET / HTTP/2
> Host: api.github.com
> User-Agent: curl/8.5.0
> Accept: */*
> 
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* old SSL session ID is stale, removing
< HTTP/2 200 
< date: Thu, 19 Dec 2024 15:36:06 GMT
< cache-control: public, max-age=60, s-maxage=60
< vary: Accept,Accept-Encoding, Accept, X-Requested-With
< x-github-api-version-selected: 2022-11-28
< access-control-expose-headers: ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset
< access-control-allow-origin: *
< strict-transport-security: max-age=31536000; includeSubdomains; preload
< x-frame-options: deny
< x-content-type-options: nosniff
< x-xss-protection: 0
< referrer-policy: origin-when-cross-origin, strict-origin-when-cross-origin
< content-security-policy: default-src 'none'
< server: github.com
< content-type: application/json; charset=utf-8
< x-github-media-type: github.v3; format=json
< etag: W/"4f825cc84e1c733059d46e76e6df9db557ae5254f9625dfe8e1b09499c449438"
< x-ratelimit-limit: 60
< x-ratelimit-remaining: 59
< x-ratelimit-reset: 1734626184
< x-ratelimit-resource: core
< x-ratelimit-used: 1
< accept-ranges: bytes
< content-length: 2262
< x-github-request-id: C67E:20A1BD:3202E14:631734B:67643D78
< 
{"current_user_url":"https://api.github.com/user","current_user_authorizations_html_url":"https://github.com/settings/connections/applications{/client_id}","authorizations_url":"https://api.github.com/authorizations","code_search_url":"https://api.github.com/search/code?q={query}{&page,per_page,sort,order}","commit_search_url":"https://api.github.com/search/commits?q={query}{&page,per_page,sort,order}","emails_url":"https://api.github.com/user/emails","emojis_url":"https://api.github.com/emojis","events_url":"https://api.github.com/events","feeds_url":"https://api.github.com/feeds","followers_url":"https://api.github.com/user/followers","following_url":"https://api.github.com/user/following{/target}","gists_url":"https://api.github.com/gists{/gist_id}","hub_url":"https://api.github.com/hub","issue_search_url":"https://api.github.com/search/issues?q={query}{&page,per_page,sort,order}","issues_url":"https://api.github.com/issues","keys_url":"https://api.github.com/user/keys","label_search_url":"https://api.github.com/search/labels?q={query}&repository_id={repository_id}{&page,per_page}","notifications_url":"https://api.github.com/notifications","organization_url":"https://api.github.com/orgs/{org}","organization_repositories_url":"https://api.github.com/orgs/{org}/repos{?type,page,per_page,sort}","organization_teams_url":"https://api.github.com/orgs/{org}/teams","public_gists_url":"https://api.github.com/gists/public","rate_limit_url":"https://api.github.com/rate_limit","repository_url":"https://api.github.com/repos/{owner}/{repo}","repository_search_url":"https://api.github.com/search/repositories?q={query}{&page,per_page,sort,order}","current_user_repositories_url":"https://api.github.com/user/repos{?type,page,per_page,sort}","starred_url":"https://api.github.com/user/starred{/owner}{/repo}","starred_gists_url":"https://api.github.com/gists/starred","topic_search_url":"https://api.github.com/search/topics?q={query}{&page,per_page}","user_url":"https://api.github.com/users/{user}","user_organizations_url":"https:/* Connection #0 to host api.github.com left intact
/api.github.com/user/orgs"user}/repos{?type,page,perearch/users?q={query}{&pag
areyeslo@QLK333KY23473FB:~/Github$ curl -v https://api.individual.githubcopilot.com/_ping
* Host api.individual.githubcopilot.com:443 was resolved.
* IPv6: (none)
* IPv4: 140.82.114.22
*   Trying 140.82.114.22:443...
* Connected to api.individual.githubcopilot.com (140.82.114.22) port 443
* ALPN: curl offers h2,http/1.1
* TLSv1.3 (OUT), TLS handshake, Client hello (1):
*  CAfile: /etc/ssl/certs/ca-certificates.crt
*  CApath: /etc/ssl/certs
* TLSv1.3 (IN), TLS handshake, Server hello (2):
* TLSv1.3 (IN), TLS handshake, Encrypted Extensions (8):
* TLSv1.3 (IN), TLS handshake, Certificate (11):
* TLSv1.3 (IN), TLS handshake, CERT verify (15):
* TLSv1.3 (IN), TLS handshake, Finished (20):
* TLSv1.3 (OUT), TLS change cipher, Change cipher spec (1):
* TLSv1.3 (OUT), TLS handshake, Finished (20):
* SSL connection using TLSv1.3 / TLS_AES_128_GCM_SHA256 / X25519 / RSASSA-PSS
* ALPN: server accepted h2
* Server certificate:
*  subject: CN=*.individual.githubcopilot.com
*  start date: May  8 00:00:00 2024 GMT
*  expire date: May  8 23:59:59 2025 GMT
*  subjectAltName: host "api.individual.githubcopilot.com" matched cert's "*.individual.githubcopilot.com"
*  issuer: C=GB; ST=Greater Manchester; L=Salford; O=Sectigo Limited; CN=Sectigo RSA Domain Validation Secure Server CA
*  SSL certificate verify ok.
*   Certificate level 0: Public key type RSA (4096/152 Bits/secBits), signed using sha256WithRSAEncryption
*   Certificate level 1: Public key type RSA (2048/112 Bits/secBits), signed using sha384WithRSAEncryption
*   Certificate level 2: Public key type RSA (4096/152 Bits/secBits), signed using sha384WithRSAEncryption
* using HTTP/2
* [HTTP/2] [1] OPENED stream for https://api.individual.githubcopilot.com/_ping
* [HTTP/2] [1] [:method: GET]
* [HTTP/2] [1] [:scheme: https]
* [HTTP/2] [1] [:authority: api.individual.githubcopilot.com]
* [HTTP/2] [1] [:path: /_ping]
* [HTTP/2] [1] [user-agent: curl/8.5.0]
* [HTTP/2] [1] [accept: */*]
> GET /_ping HTTP/2
> Host: api.individual.githubcopilot.com
> User-Agent: curl/8.5.0
> Accept: */*
> 
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* old SSL session ID is stale, removing
< HTTP/2 200 
< date: Thu, 19 Dec 2024 15:36:36 GMT
< content-length: 2
< content-type: text/plain; charset=utf-8
< x-github-backend: Kubernetes
< x-github-request-id: C68E:20CF5D:19A29A8:21190D4:67643D84
< 
* Connection #0 to host api.individual.githubcopilot.com left intact
OKareyeslo@QLK333KY23473FB:~/Github$ 

On Github Authentication Logs:
2024-12-19 08:51:03.836 [info] Reading sessions from keychain...
2024-12-19 08:51:03.837 [info] Getting sessions for all scopes...
2024-12-19 08:51:03.837 [info] Getting sessions for all scopes...
2024-12-19 08:51:03.837 [info] Getting sessions for all scopes...
2024-12-19 08:51:03.895 [info] Got stored sessions!
2024-12-19 08:51:03.895 [info] Got 2 verified sessions.
2024-12-19 08:51:03.896 [info] Got 2 sessions for ...
2024-12-19 08:51:03.896 [info] Got 2 sessions for ...
2024-12-19 08:51:03.896 [info] Got 2 sessions for ...
2024-12-19 08:51:04.117 [info] Getting sessions for all scopes...
2024-12-19 08:51:04.119 [info] Got 2 sessions for ...
2024-12-19 08:51:04.120 [info] Getting sessions for all scopes...
2024-12-19 08:51:04.120 [info] Got 2 sessions for ...
2024-12-19 08:51:04.120 [info] Getting sessions for all scopes...
2024-12-19 08:51:04.120 [info] Got 2 sessions for ...
2024-12-19 08:51:04.121 [info] Getting sessions for all scopes...
2024-12-19 08:51:04.121 [info] Got 2 sessions for ...
2024-12-19 08:51:04.121 [info] Getting sessions for all scopes...
2024-12-19 08:51:04.121 [info] Got 2 sessions for ...
2024-12-19 08:51:39.025 [info] Getting sessions for read:user,repo,user:email,workflow...
2024-12-19 08:51:39.025 [info] Got 1 sessions for read:user,repo,user:email,workflow...
2024-12-19 08:51:39.599 [info] Getting sessions for read:user,repo,user:email,workflow...
2024-12-19 08:51:39.600 [info] Got 1 sessions for read:user,repo,user:email,workflow...
2024-12-19 08:52:00.857 [info] Reading sessions from keychain...
2024-12-19 08:52:00.857 [info] Getting sessions for all scopes...
2024-12-19 08:52:00.857 [info] Getting sessions for all scopes...
2024-12-19 08:52:00.857 [info] Getting sessions for all scopes...
2024-12-19 08:52:00.959 [info] Got stored sessions!
2024-12-19 08:52:00.959 [info] Got 2 verified sessions.
2024-12-19 08:52:00.959 [info] Got 2 sessions for ...
2024-12-19 08:52:00.959 [info] Got 2 sessions for ...
2024-12-19 08:52:00.959 [info] Got 2 sessions for ...
2024-12-19 08:52:01.184 [info] Getting sessions for all scopes...
2024-12-19 08:52:01.184 [info] Got 2 sessions for ...
2024-12-19 08:52:01.184 [info] Getting sessions for all scopes...
2024-12-19 08:52:01.184 [info] Got 2 sessions for ...
2024-12-19 08:52:01.184 [info] Getting sessions for all scopes...
2024-12-19 08:52:01.184 [info] Got 2 sessions for ...
2024-12-19 08:52:01.184 [info] Getting sessions for all scopes...
2024-12-19 08:52:01.184 [info] Got 2 sessions for ...
2024-12-19 08:52:01.185 [info] Getting sessions for all scopes...
2024-12-19 08:52:01.185 [info] Got 2 sessions for ...
2024-12-19 08:52:03.582 [info] Getting sessions for all scopes...
2024-12-19 08:52:03.583 [info] Got 2 sessions for ...
2024-12-19 08:52:03.600 [info] Getting sessions for read:user,repo,user:email,workflow...
2024-12-19 08:52:03.601 [info] Got 1 sessions for read:user,repo,user:email,workflow...
2024-12-19 08:52:04.340 [info] Getting sessions for read:user,repo,user:email,workflow...
2024-12-19 08:52:04.340 [info] Got 1 sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:39.055 [info] Getting sessions for all scopes...
2024-12-19 08:53:39.055 [info] Got 2 sessions for ...
2024-12-19 08:53:39.055 [info] Getting sessions for all scopes...
2024-12-19 08:53:39.055 [info] Got 2 sessions for ...
2024-12-19 08:53:39.056 [info] Getting sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:39.056 [info] Got 1 sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:39.079 [info] Getting sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:39.079 [info] Got 1 sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:39.324 [info] Getting sessions for all scopes...
2024-12-19 08:53:39.324 [info] Got 2 sessions for ...
2024-12-19 08:53:39.374 [info] Getting sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:39.374 [info] Got 1 sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:39.428 [info] Getting sessions for all scopes...
2024-12-19 08:53:39.429 [info] Got 2 sessions for ...
2024-12-19 08:53:39.515 [info] Getting sessions for all scopes...
2024-12-19 08:53:39.516 [info] Got 2 sessions for ...
2024-12-19 08:53:39.516 [info] Getting sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:39.516 [info] Got 1 sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:39.642 [info] Getting sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:39.643 [info] Got 1 sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:39.643 [info] Getting sessions for all scopes...
2024-12-19 08:53:39.643 [info] Got 2 sessions for ...
2024-12-19 08:53:39.695 [info] Getting sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:39.695 [info] Got 1 sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:40.612 [info] Getting sessions for all scopes...
2024-12-19 08:53:40.612 [info] Got 2 sessions for ...
2024-12-19 08:53:40.612 [info] Getting sessions for all scopes...
2024-12-19 08:53:40.612 [info] Got 2 sessions for ...
2024-12-19 08:53:40.658 [info] Getting sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:40.659 [info] Got 1 sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:40.827 [info] Getting sessions for all scopes...
2024-12-19 08:53:40.827 [info] Got 2 sessions for ...
2024-12-19 08:53:40.834 [info] Getting sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:40.834 [info] Got 1 sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:41.150 [info] Getting sessions for all scopes...
2024-12-19 08:53:41.150 [info] Got 2 sessions for ...
2024-12-19 08:53:41.223 [info] Getting sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:41.223 [info] Got 1 sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:41.945 [info] Getting sessions for all scopes...
2024-12-19 08:53:41.945 [info] Got 2 sessions for ...
2024-12-19 08:53:41.965 [info] Getting sessions for read:user,repo,user:email,workflow...
2024-12-19 08:53:41.965 [info] Got 1 sessions for read:user,repo,user:email,workflow...

On Github Copilot logs:
2024-12-19 08:53:39.544 [info] [certificates] Removed 1 expired certificates
2024-12-19 08:53:40.213 [info] [fetcher] Using Helix fetcher, Electron fetcher is not available.
2024-12-19 08:53:40.213 [info] [code-referencing] Public code references are enabled.

On Github Copilot Chat:
2024-12-19 08:53:39.493 [info] Can't use the Electron fetcher in this environment.
2024-12-19 08:53:39.493 [info] Using the Node fetch fetcher.
2024-12-19 08:53:39.493 [info] Initializing Git extension service.
2024-12-19 08:53:39.493 [info] Successfully activated the vscode.git extension.
2024-12-19 08:53:39.493 [info] Enablement state of the vscode.git extension: true.
2024-12-19 08:53:39.493 [info] Successfully registered Git commit message provider.
2024-12-19 08:53:39.979 [info] Logged in as areyeslo
2024-12-19 08:53:39.947 [info] Got Copilot token for areyeslo
2024-12-19 08:53:40.386 [info] copilot token chat_enabled: true, sku: trial_30_monthly_subscriber
2024-12-19 08:53:40.386 [info] GitHub.vscode-pull-request-github extension is not yet activated.
2024-12-19 08:53:40.401 [info] Registering default platform agent...
2024-12-19 08:53:40.983 [info] Fetched model metadata in 1025ms 5f72993c-1d24-4a3a-8735-c9c7971a8b27
2024-12-19 08:53:40.985 [info] activationBlocker from 'languageModelAccess' took for 2061ms

Then, I closed VSCode and open again. I cannot access Github Copilot.

I still have connection to api.github.com

Extension version: 0.23.2
VS Code version: Code 1.96.1 (42b266171e51a016313f47d0c48aca9295b9cbb2, 2024-12-17T17:50:05.206Z)
OS version: Windows_NT x64 10.0.22631
Modes:
Remote OS version: Linux x64 5.15.167.4-microsoft-standard-WSL2

<details><summary>Prompt Details</summary>
<pre>
Info: Can't use the Electron fetcher in this environment.
Info: Using the Node fetch fetcher.
Info: Initializing Git extension service.
Info: Successfully activated the vscode.git extension.
Info: Enablement state of the vscode.git extension: true.
Info: Successfully registered Git commit message provider.
Debug: [context keys] Updating context keys.
</pre>
</details><details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|12th Gen Intel(R) Core(TM) i7-1265U (12 x 2688)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|31.83GB (15.76GB free)|
|Process Argv|--crash-reporter-id e6990928-de2c-4cff-a147-afa8eecc5dd7|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu|
|OS|Linux x64 5.15.167.4-microsoft-standard-WSL2|
|CPUs|12th Gen Intel(R) Core(TM) i7-1265U (12 x 0)|
|Memory (System)|7.76GB (5.68GB free)|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt551cf:31179979
vscod805cf:30301675
binariesv615:30325510
vsaa593:30376534
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
2i9eh265:30646982
962ge761:30959799
pythonnoceb:30805159
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
dvdeprecation:31068756
dwnewjupytercf:31046870
newcmakeconfigv2:31071590
nativerepl1:31139838
pythonrstrctxt:31112756
nativeloc1:31192215
cf971741:31144450
iacca1:31171482
notype1cf:31157160
5fd0e150:31155592
dwcopilot:31170013
stablechunks:31184530
6074i472:31201624

```

</details>

**Comment by bpasero** [comment]
This is likely a duplicate of https://github.com/microsoft/vscode-copilot-release/issues/3375

**Comment by areyeslo** [comment]
the issue you mentioned is already closed. I am using the last version of Github copilot extension in VSCode and it is not working.

**Comment by bpasero** [comment]
Yeah, a fix will go out later today.

**Comment by joyceerhl** [comment]
This should be fixed in the 1.96.2 release.

**Comment by lastboy1228** [comment]
update to 1.96.2, and disable NodeFetch
```
    "github.copilot.advanced": {
        "debug.useElectronFetcher": false,
        "debug.useNodeFetcher": true,
        "debug.useNodeFetchFetcher": false,
    },
```
